app/Http/Controllers/DashboardController.php

```diff
@@ -36,4 +45,46 @@ private function getTopLoadouts()
 
         return $loadouts;
     }
+
+    private function getRecentTopLoadouts()
```

Comment:
I wonder if we change it up a bit. We could change this to be character agnostic. Like, let's just put the top 10 loadouts up from any class for last 2 weeks. That will kind of break up the colors and kind of ... predictable nature of the dashboard without re-working it majorly.